examples/cookie-game/src/index.ts

```diff
@@ -0,0 +1,336 @@
+        case actions.incrementTimeMachine: {
+            const updatedCookieBaker = addTimeMachine(cookieBaker);
+
+            //action successful, update state
+            sourceValue.cookieBaker = updatedCookieBaker;
+            console.log("Successfully minted time machine");
+            saveState(source, sourceValue);
+            break;
+        }
+        case actions.incrementAntimatter: {
+            const updatedCookieBaker = addAntimatter(cookieBaker);
+
+            //action successful, update state
+            sourceValue.cookieBaker = updatedCookieBaker;
+            console.log("Successfully minted antimatter");
+            saveState(source, sourceValue);
+            break;
+        }
+        case actions.incrementPrism: {
+            const updatedCookieBaker = addPrism(cookieBaker);
+
+            //action successful, update state
+            sourceValue.cookieBaker = updatedCookieBaker;
+            console.log("Successfully minted prism");
+            saveState(source, sourceValue);
+            break;
+        }
+        case actions.incrementChanceMaker: {
+            const updatedCookieBaker = addChanceMaker(cookieBaker);
+
+            //action successful, update state
+            sourceValue.cookieBaker = updatedCookieBaker;
+            console.log("Successfully minted chance maker");
+            saveState(source, sourceValue);
+            break;
+        }
+        case actions.incrementFractal: {
+            const updatedCookieBaker = addFractal(cookieBaker);
+
+            //action successful, update state
+            sourceValue.cookieBaker = updatedCookieBaker;
+            console.log("Successfully minted fractal");
+            saveState(source, sourceValue);
+            break;
+        }
+        case actions.incrementJavascript: {
+            const updatedCookieBaker = addJavascript(cookieBaker);
+
+            //action successful, update state
+            sourceValue.cookieBaker = updatedCookieBaker;
+            console.log("Successfully minted javascript");
+            saveState(source, sourceValue);
+            break;
+        }
+        case actions.incrementIdleverse: {
+            const updatedCookieBaker = addIdleverse(cookieBaker);
+
+            //action successful, update state
+            sourceValue.cookieBaker = updatedCookieBaker;
+            console.log("Successfully minted idleverse");
+            saveState(source, sourceValue);
+            break;
+        }
+        case actions.incrementCordex: {
+            const updatedCookieBaker = addCordex(cookieBaker);
+
+            //action successful, update state
+            sourceValue.cookieBaker = updatedCookieBaker;
+            console.log("Successfully minted cordex");
+            saveState(source, sourceValue);
+            break;
+        }
+    }
+}
+
+main(
+    //tz address must be replaced by a correct one obtained with 
+    //deku-cli create-wallet
+    {
+        "tz1VULT8pu1NoWs7YPFWuvXSg3JSdGq55TXc":
```

Comment:
We need to rework this default state. It can't use a default address. I guess the only correct choice is an empty one.
Then, we need to manage the address here. Maybe in the payload? Like `{"address":"tzxxxxxx", "action":"cookie"}`
What do you think?